requirements/prod.txt

```diff
@@ -382,10 +382,9 @@ mohawk==1.1.0 \
 basket-client==1.0.0 \
     --hash=sha256:fee512c8deabde53aa33b08fa34a992de3085e781be192c4120adb086d0d20c1 \
     --hash=sha256:f9f9ff4870cc1d171ff9f00a8d968336d0e74178f5ac1612bc85be39a3f0cc2c
-typing==3.7.4.1 \
-    --hash=sha256:91dfe6f3f706ee8cc32d38edbbf304e9b7583fb37108fef38229617f8b3eba23 \
-    --hash=sha256:c8cabb5ab8945cd2f54917be357d134db9cc1eb039e59d1606dc1e60cb1d9d36 \
-    --hash=sha256:f38d83c5a7a7086543a0f649564d661859c5146a85775ab90c0d2f93ffaa9714
+typing==3.7.4.3 \
```

Comment:
You removed it from `docs.txt` but it's still there.

Reply:
missed that https://github.com/mozilla/addons-server/pull/14928/commits/7a030fee6d84d991b95b8ecd9c62d32781a4a9e7